tests/conftest.py

```diff
@@ -0,0 +1,181 @@
+"""Pytest utility classes, functions and fixtures"""
```

Comment:
Missing . and need a blank line after the module docstring? (I think? it would be good to get DeepSource back on track for these things).